.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -29,9 +29,10 @@ body:
       render: shell
     validations:
       required: true
-  - type: input
+  - type: textarea
     attributes:
-      label: SciPy/NumPy/Python version information
-      description: 'Please run the following and paste the result in a code block - `import scipy;scipy.show_config()` or on older versions `import sys, scipy, numpy; print(scipy.__version__, numpy.__version__, sys.version_info)`'
+      label: SciPy/NumPy/Python version and system information
+      description: 'Please run the following and paste the result in a code block - `import sys, scipy, numpy; print(scipy.__version__, numpy.__version__, sys.version_info); scipy.show_config()`'
```

Comment:
```suggestion
      description: 'Please run the following and paste the result here - `import sys, scipy, numpy; print(scipy.__version__, numpy.__version__, sys.version_info); scipy.show_config()`'
```
It's in a code block by default.

One other optimization: can we put the code to run on a separate line? This is what it looked like:
![image](https://user-images.githubusercontent.com/6570539/222206605-31bd2e56-0eac-4640-a7b7-6435c94fc275.png)

And I think that this would make it all fit on one line so it would be easier to copy-paste the whole thing. Otherwise, I imagine the last command might be left behind.

Reply:
OK.

Reply:
I checked their doc and syntax doc and don't see mention of that either https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository